OpenRA.Mods.Common/UtilityCommands/CheckRuntimeAssembliesCommand.cs

```diff
@@ -0,0 +1,58 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2019 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace OpenRA.Mods.Common.UtilityCommands
+{
+	public class CheckRuntimeAssembliesCommand : IUtilityCommand
+	{
+		string IUtilityCommand.Name { get { return "--check-runtime-assemblies"; } }
+
+		bool IUtilityCommand.ValidateArguments(string[] args)
+		{
+			return true;
+		}
+
+		[Desc("Generates an error if any assembly filenames in the current domain don't match the given whitelist",
```

Comment:
I don't find the whitelist doc here to be helpful by itself.

What format is `WHITELIST` expected to be in?

A comma-separated collection of assembly filenames? Filepaths? Can there be spaces in after commas? Etc.

I'm probably being picky but I'm hopefully that verbose docs will reduce the number of repetitive questions the active maintainers field.